templates/default/sql/dml/3.2.0.sql.erb

```diff
@@ -6,4 +6,28 @@ REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("git_image_version",
 
 REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("testconnector_image_version", "<%= node['hops']['docker']['testconnector']['image']['version'] %>");
 
-REPLACE INTO `hopsworks`.`variables`(`id`, `value`, `visibility`) VALUES ("enable_flyingduck", "<%= node['hopsworks']['enable_flyingduck'] %>", 1);
\ No newline at end of file
+REPLACE INTO `hopsworks`.`variables`(`id`, `value`, `visibility`) VALUES ("enable_flyingduck", "<%= node['hopsworks']['enable_flyingduck'] %>", 1);
+
+-- Move Grants to DML files 
+-- Grants from hopsworks/default.rb 
+
+GRANT NDB_STORED_USER ON *.* TO '<%= node['hopsworks']['mysql']['user'] %>'@'127.0.0.1';
+GRANT ALL PRIVILEGES ON <%= node['hopsworks']['db'] %>.* TO '<%= node['hopsworks']['mysql']['user']%>'@'127.0.0.1';
+GRANT SELECT ON <%= node['hops']['db'] %>.* TO '<%= node['hopsworks']['mysql']['user']%>'@'127.0.0.1';
+GRANT ALL PRIVILEGES ON <%= node['hops']['db'] %>.yarn_projects_quota TO '<%= node['hopsworks']['mysql']['user'] %>'@'127.0.0.1';
+GRANT ALL PRIVILEGES ON <%= node['hops']['db'] %>.hdfs_directory_with_quota_feature TO '<%= node['hopsworks']['mysql']['user'] %>'@'127.0.0.1';
+GRANT SELECT ON metastore.* TO '<%= node['hopsworks']['mysql']['user'] %>'@'127.0.0.1';
+
+-- Grants for Kafka 
+CREATE USER IF NOT EXISTS '<%= node['kkafka']['mysql']['user'] %>'@'127.0.0.1' IDENTIFIED BY '<%= node['kkafka']['mysql']['password'] %>';
+REVOKE ALL PRIVILEGES ON *.* FROM '<%= node['kkafka']['mysql']['user'] %>'@'127.0.0.1';
```

Comment:
This should to go to the update, right?

Reply:
I guess you mean upgrade? If so, yes we always run all the DML files on upgrade

Reply:
Yeah sorry my bad, I was confused